src/main/java/hillel/doctorRest/clinic/schedule/Schedule.java

```diff
@@ -18,6 +18,8 @@
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Integer id;
+    @Version
+    private Integer version;
```

Comment:
As we discussed, using optimistic lock makes no sense here, the DB constraint does all the job